pkg/commands/command_helper.go

```diff
@@ -164,7 +164,7 @@ func (ch CommandHelper) Printlnf(format string, args ...interface{}) error {
 }
 
 func (ch CommandHelper) OutOrErrWriter() io.Writer {
-	if ch.output {
+	if ch.output { // this is intense
```

Comment:
// this is intense?

Reply:
lol i was typing that in slack one day (or so i thought) and couldnt figure out where the text went, guess you found it